docs/quickstart.rst

```diff
@@ -31,13 +31,18 @@ Quickstart
 
 7. Open up an elite programming editor and type::
 
+       #! /usr/bin/env hy
```

Comment:
Force of habit. The space is there so it lines up nicely with comments below it (if any):

```hy
#! /usr/bin/env hy
## stuff.hy -- does stuff
## Copyright (C) 2015 Some One <over@the.rainbow>
```

Compare it to the spaceless variant:

```hy
#!/usr/bin/env hy
## stuff.hy -- does stuff
## Copyright (C) 2015 Some One <over@the.rainbow>
```

Striking difference in favour of the first, I say!